README.md

```diff
@@ -37,6 +37,209 @@
 * Can work with Extensions
 * Can work with making a Class and Enums
 
+## JF NEW TEXT
+Just as in any object-oriented programming language, classes in Swift encapsulate data and functionality. In practical terms, that means properties that are variables (`var`) and constants (`let`) as well as methods. (Class methods are functions that are declared within a class. Functions can also be declared outside of classes.)
+
+>For more on classes insert Xref to Classes
+
+## Create a New Class - Person
+You can create a Person class.
+
+Encapsulated in that class will be three properties: name, height, and favoriteFood. Here are the property declarations:
+
+		var name: String
+		var height: Double
+		var favoriteFood: String
+
```

Comment:
Thanks. Where are we now with this — do you want me to make changes or you do? And for the future, doyou want me to create a new file or work in the ReadMe? 

> On Jul 5, 2016, at 4:46 PM, Jim Campagno <notifications@github.com <mailto:notifications@github.com>> wrote:
> 
> In README.md <https://github.com/learn-co-curriculum/swift-protocols-readme/pull/1#discussion_r69634934>:
> 
> > @@ -37,6 +37,209 @@
> >  * Can work with Extensions
> >  * Can work with making a Class and Enums
> >  
> > +## JF NEW TEXT
> > +Just as in any object-oriented programming language, classes in Swift encapsulate data and functionality. In practical terms, that means properties that are variables (`var`) and constants (`let`) as well as methods. (Class methods are functions that are declared within a class. Functions can also be declared outside of classes.)
> > +
> > +>For more on classes insert Xref to Classes
> > +
> > +## Create a New Class - Person
> > +You can create a Person class.
> > +
> > +Encapsulated in that class will be three properties: name, height, and favoriteFood. Here are the property declarations:
> > +
> > +		var name: String
> > +		var height: Double
> > +		var favoriteFood: String
> > +
> A prettier way this can be represented in markdown (referring to the code snippet), is doing so like this:
> 
>  <https://camo.githubusercontent.com/0be82af351118ffa23d3ac18cd8b73af230f62d1/68747470733a2f2f692e696d6775722e636f6d2f6a53764964354e2e706e67>
> Doing so will produce the following outcome, notice the colors!
> 
> var name: String
> var height: Double
> var favoriteFood: String
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub <https://github.com/learn-co-curriculum/swift-protocols-readme/pull/1/files/878ea6cede03f965e724bbb71e2deb103869da8d#r69634934>, or mute the thread <https://github.com/notifications/unsubscribe/ABImsssTHXWuwK86LYoR4RBrhaHHWSfmks5qSsK-gaJpZM4JFgoF>.
>